internal/testrunner/runners/system/servicedeployer/terraform_env.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package servicedeployer
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	awsAccessKeyID     = "AWS_ACCESS_KEY_ID"
+	awsSecretAccessKey = "AWS_SECRET_ACCESS_KEY"
+	awsProfile         = "AWS_PROFILE"
+	awsRegion          = "AWS_REGION"
```

Comment:
It would be really nice if we could keep any mention of integration-specific variables/constants out of the elastic-package source code. I know there is [another place](https://github.com/elastic/elastic-package/pull/227/files#r565746042) where this is happening as well. Similar to the other place, could you please add a TODO comment here with a link to an issue for cleaning this up?

Reply:
https://github.com/elastic/elastic-package/issues/235